resources/views/dimmers.blade.php

```diff
@@ -1,7 +1,17 @@
 @php
 $dimmers = \Arrilot\Widgets\Facade::group('voyager::dimmers');
-$count = $dimmers->count();
 
+// Loop over the defined dashboard widgets. Get the related model and remove the
+// widget from the group if the current user hasn't the permission to read it.
+foreach (config('voyager.dashboard.widgets') as $widgetClass) {
+    $relatedModel = app($widgetClass)->getRelatedModel();
+
+    if (! Auth::user()->can('read', $relatedModel)) {
```

Comment:
I think since widgets give a count and are linked to the listing page browse permission should be checked instead of read.

Reply:
Oo ##&@*. Good catch!